### ISSUE TITLE
Release binaries can't find shared libraries

### ISSUE THREAD

**Issue by jcol2** [jcol2/midistar#8]
Release binaries (for both v0.1.0 and v0.2.0) cannot find shared libraries when the `./run` command is executed.

On Linux:
```
 jez@sunnyvale:~/sandbox/midistar-0.2.0$ ./run 
./build/bin/midistar: error while loading shared libraries: librtmidi.so.4: cannot open shared object file: No such file or directory
```

On OSX: 
```
Jeremys-MBP:midistar-0.2.0 jez$ ./run 
dyld: Library not loaded: @rpath/libsfml-graphics.2.4.dylib
  Referenced from: /Users/jez/Desktop/midistar-0.2.0/build/bin/midistar
  Reason: image not found
Trace/BPT trap: 5
```

**Comment by jcol2** [comment]
It appears that CMake links to shared libraries with absolute paths, so if the midistar project folder is moved (e.g. if a release is downloaded to a different path), the libraries will not be found.

**Comment by jcol2** [comment]
The issue arises when midistar shared libraries are at a different path during runtime vs building / linking. As release binaries are downloaded on to different machines in various locations, this error is likely to occur.

A solution is to set system environment variables in the `run` script to help the system find the shared libraries.

**Comment by jcol2** [comment]
Fixed in commit d1fb561.
